aws/dms/cron-project-hierarchy.json

```diff
@@ -0,0 +1,48 @@
+{
+  "rules": [
+    {
+      "rule-type": "selection",
+      "rule-id": "1",
+      "rule-name": "1",
+      "object-locator": {
+        "schema-name": "pegasus",
+        "table-name": "storage_apps"
+      },
+      "rule-action": "include"
+    },
+    {
+      "rule-type": "selection",
+      "rule-id": "2",
+      "rule-name": "2",
+      "object-locator": {
+        "schema-name": "pegasus",
+        "table-name": "user_storage_ids"
+      },
+      "rule-action": "include"
+    },
+    {
+      "rule-type": "transformation",
+      "rule-id": "3",
+      "rule-name": "3",
+      "rule-target": "column",
+      "object-locator": { 
+        "schema-name": "pegasus",
+        "table-name": "storage_apps",
+        "column-name": "value"
+      },
+      "rule-action": "remove-column"
```

Comment:
It seems that (1) is the path of least resistance. Especially as I envision it might be useful to have an index on this (new) column in the production DB.